ogr/services/mock/github_mock.py

```diff
@@ -0,0 +1,21 @@
+# mocking github interface
+import github
+from ogr.services.mock import core
+
+
+class MockGitHubProject(core.MockClass):
```

Comment:
Let's make it more general:
```suggestion
class MockProject(core.MockClass):
```

Reply:
And mock the `AbstractService`/`AbstractProject`/`AbstractUser`...

Reply:
Yep, we can discuss then what everything we need to mock. Maybe we can strast with some minimal mock and them extend it.
 rename it to ``MockProject`` but this is mocking github project, and in case ``libpagure`` has also project, it could have another methods, so checking method names them will not work

Reply:
I would like to have it as another implementation of the `AbstractProject`. All implementation (`GithubProject`/`PagureProject`) should implement the same methods -- this is the purpose of the OGR: One API for all git-forges.